instance/models/instance.py

```diff
@@ -139,17 +143,37 @@ def active_server_set(self):
         return self.server_set.exclude_terminated()
 
     @property
-    def status(self):
+    def current_server(self):
         """
-        Instance status
+        Current active server. Raises InconsistentInstanceState if more than
+        one exists.
         """
         active_server_set = self.active_server_set
         if not active_server_set:
-            return self.EMPTY
+            return
         elif active_server_set.count() > 1:
             raise InconsistentInstanceState('Multiple servers are active, which is unsupported')
-        else:
-            return active_server_set[0].status
+        return active_server_set[0]
```

Comment:
It has the same effect, but I think you are right, that's not the point of this PR. I will revert.